### ISSUE TITLE
Update requirements table with MAUR information

### ISSUE THREAD

**Issue by cconcolato** [w3c/dapt-reqs#5]
ADPT had [[media-accessibility-reqs](https://w3c.github.io/dapt-reqs/#bib-media-accessibility-reqs)] info in the table. It should be added.

**Comment by nigelmegitt** [comment]
I'll do this.
